Nikita_Konkov_Application/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,84 @@
+﻿
+using System.Diagnostics;
+using System.Reflection.Metadata.Ecma335;
+
+namespace CSharpException
+{
+    internal class Triangle
+    {
+        private int _firstSide;
+        private int _secondSide;
+        private int _thirdSide;
+        
+        public Triangle() { }
+
+        public int FirstSide
+        {
+            get => _firstSide;
+            set
+            {
+                _firstSide = value;
+            }
+        }
+        public int SecondSide 
+        { 
+            get => _secondSide;
+            set
+            {
+                value = ZeroCheck(SecondSide);
+                _secondSide = value;
+            }
+        }      
+
+        public int ThirdSide
+        {
+            get => _thirdSide;
+            set
+            {
+                _thirdSide = value;
+            }
+        }
+
+        public int SideCheck(int sideOne, int sideTwo, int sideThree)
+        {
+            bool oneBiger = sideOne > sideTwo + sideThree;
+            bool twoBiger = sideTwo > sideOne + sideTwo;
+            bool threeBiger = sideThree > sideTwo + sideOne;
+
+            if (oneBiger || twoBiger || threeBiger)
+            {
+                throw new OversizedTriangeSideException("One of sides is longer then the sum of the other two!");
+            }
+            else
+            {
+                return sideOne;
+                return sideTwo;
+                return sideThree;
+            }
+        }
+
+        public int ZeroCheck(int justSide)
+        {
+            while (justSide == 0)
+            {
+                justSide = int.Parse(Console.ReadLine());
```

Comment:
the method should only check if the number zero or not. You should implement number input in Program.cs and reenter the number in Program.cs.